chia/full_node/mempool_manager.py

```diff
@@ -150,40 +175,16 @@ def always(mm: MempoolManager, mi: MempoolItem) -> bool:
 
             item_inclusion_filter = always
 
-        cost_sum = 0  # Checks that total cost does not exceed block maximum
-        fee_sum = 0  # Checks that total fees don't exceed 64 bits
-        spend_bundles: List[SpendBundle] = []
-        removals = []
-        additions = []
-        broke_from_inner_loop = False
         log.info(f"Starting to make block, max cost: {self.constants.MAX_BLOCK_COST_CLVM}")
-        for dic in reversed(self.mempool.sorted_spends.values()):
-            if broke_from_inner_loop:
-                break
-            for item in dic.values():
-                log.info(f"Cumulative cost: {cost_sum}, fee per cost: {item.fee / item.cost}")
-                if (
-                    item.cost + cost_sum <= self.limit_factor * self.constants.MAX_BLOCK_COST_CLVM
-                    and item.fee + fee_sum <= self.constants.MAX_COIN_AMOUNT
-                    and item_inclusion_filter(self, item)
```

Comment:
I'm pretty sure the inclusion filter is meant to filter out single items, but the way it's used here (and still in the function you factored out) it will abort the whole loop. I believe this is wrong. We need a test for this also.

Reply:
@aqk could you take a look? ^^

Reply:
@aqk I sent a [fix suggestion](https://github.com/Chia-Network/chia-blockchain/pull/13981/commits/0b64ed4286a5bf2ad7530fba3b1ed7a8aec75475) if you could kindly check. Thanks.

Reply:
Thanks - see you in a few mins

Reply:
Do you want to merge https://github.com/Chia-Network/chia-blockchain/pull/13981/commits/0b64ed4286a5bf2ad7530fba3b1ed7a8aec75475 first?

Reply:
Okay, we decided to put in the small fix first, and do more work on tests on this PR.

Reply:
PR #13992 ^